server/router.ts

```diff
@@ -144,15 +144,23 @@ function setupRouter(
   const router = express.Router();
 
   router.use((req, res, next) => {
+    console.log('requested path', req.path)
```

Comment:
can we use log.debug here rather than console.log so we can get timestamps as well?

Reply:
done